client/header/activity-panel/activity-header/index.js

```diff
@@ -13,17 +13,28 @@ import { EllipsisMenu, H } from '@woocommerce/components';
 
 class ActivityHeader extends Component {
 	render() {
-		const { title, className, menu } = this.props;
+		const { className, menu, subtitle, title, unreadMessages } = this.props;
 		const cardClassName = classnames(
-			'woocommerce-layout__activity-panel-header',
+			subtitle ? 'woocommerce-layout__inbox-panel-header' : 'woocommerce-layout__activity-panel-header',
```

Comment:
You can use `classnames` with logic, like

```
classnames(
    className,
    {
        'woocommerce-layout__inbox-panel-header': subtitle,
        'woocommerce-layout__activity-panel-header': ! subtitle,
    }
);
```

which might clear up the logic here a bit.

Reply:
Nice tip! Thank you @becdetat!
I changed it in the commit [0ce7d0f](https://github.com/woocommerce/woocommerce-admin/pull/4379/commits/0ce7d0fcb324a098931b477465d26dadea5f038f)